packages/bits/src/lib/date-picker/date-picker.component.less

```diff
@@ -78,10 +78,10 @@
         min-height: @nui-datepicker-popup-min-height;
         min-width: @nui-datepicker-popup-min-width;
         padding: (@nui-space-sm - @nui-space-xxs) @nui-space-sm;
-        .setCssVariable(background-color, nui-color-bg-light);
 
         table {
             outline: none;
+            background-color: transparent;
```

Comment:
Just curious, what is this change for?

Reply:
It is a fix for failed date-time-picker visual tests, which were found by Ivan.